### PR TITLE
Client setup

### PR DESCRIPTION
**Pull request by jaanonim**
Adding client part of game and editorconfig for project.

### CODE REVIEW

**Review by jbanasik — APPROVED** [review]
Looks good to me! :D

**Review by kpustelnik — CHANGES_REQUESTED** [review]
Please review suggestions - thx

File: .editorconfig
[the remark this answers is not in the document]
Comment (jaanonim) [reply .editorconfig:1]: No, this should be here. It's to assure that every editor is configured in the same way. You can read more [here](https://editorconfig.org/)

File: client/main.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import pygame
+
+pygame.init()
+screen = pygame.display.set_mode((1280, 720))
```
[the remark this answers is not in the document]
Comment (jaanonim) [reply client/main.py:4]: It's great idea. Maybe 1080x1920? What do you think? @kpustelnik

File: client/main.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+running = True
+
+while running:
+    for event in pygame.event.get():
+        if event.type == pygame.QUIT:
+            running = False
+
+    screen.fill("purple")
+    pygame.display.flip()
+    clock.tick(60)
```
[the remark this answers is not in the document]
Comment (jaanonim) [reply client/main.py:15]: Let's add it to the trello as task for later, we have for iportant stuff to do now.

File: client/main.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+pygame.init()
+screen = pygame.display.set_mode((1280, 720))
+clock = pygame.time.Clock()
+running = True
+
+while running:
+    for event in pygame.event.get():
+        if event.type == pygame.QUIT:
+            running = False
```
[the remark this answers is not in the document]
Comment (jaanonim) [reply client/main.py:11]: I don't know. I copied it from docs :-)

File: client/main.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (kpustelnik) [reply client/main.py:4]: @jaanonim Did you mean 1920x1080? I don't have vertical monitor :(

File: client/main.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (kpustelnik) [reply client/main.py:11]: @jaanonim I see. I believe we can use it for now then.

File: client/main.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (kpustelnik) [reply client/main.py:15]: Sounds good!

File: client/main.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (jaanonim) [reply client/main.py:4]: My bad, of course I meant 1920x1080.

File: .editorconfig
[the remark this answers is not in the document]
Comment (kpustelnik) [reply .editorconfig:1]: Wow that's helpful! Cool

File: client/main.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (kpustelnik) [reply client/main.py:4]: @jaanonim Let's change and I guess hte PR is ready then!

[the remark this answers is not in the document]
Comment (jaanonim) [reply client/main.py]: @kpustelnik please re-review

**Review by kpustelnik — APPROVED** [review]
That's awesome! Thx for PR!

**Pull request merged**